server/graphql_schemas/tests/attend/test_mutations.py

```diff
@@ -1,7 +1,7 @@
 import logging
 
-from django.contrib.auth.models import User
-from api.models import Attend
+# from django.contrib.auth.models import User
```

Comment:
@parseendavid Well done.... but please check commented code in line 3

Reply:
I'll remove that.